test/cql-pytest/cassandra_tests/validation/entities/uf_types_test.py

```diff
@@ -0,0 +1,345 @@
+# This file was translated from the original Java test from the Apache
+# Cassandra source repository, as of Cassandra 4.1.1 (commit f54ef5e824daf758087ff4efa94041467d46f8eb)
```

Comment:
Did you actually take the test from this specific Cassandra commit, or just copied this header from another test? :-)

Reply:
This is the actual commit I took the test from, however this isn't the commit that introduced the test or the one that most recently modified it.